docker-compose.yml

```diff
@@ -0,0 +1,12 @@
+version: '3.1'
+
+services:
+  keg-auth-pg:
+    image: postgres:12-alpine
+    container_name: keg-auth-pg
+    environment:
+      POSTGRES_DB: kegauth_tests
+      POSTGRES_HOST_AUTH_METHOD: trust
+    ports:
+    # This might not be what you want, use docker-compose.override.yml to change.
```

Comment:
https://docs.docker.com/compose/environment-variables/

We could just parameterize these things and provide default values, which would allow anyone to throw a `.env` in the folder with overrides.